app/javascript/packs/vertical_nav.js

```diff
@@ -0,0 +1,18 @@
+import { VerticalNavWrapper as VerticalNav } from 'Navigation/components/VerticalNav'
+import { safeFromJsonString } from 'utilities/json-utils'
+
+import '@patternfly/react-core/dist/styles/base.css'
+
```

Comment:
To use Open Sans, you can import it from `provider/fonts` like [here](https://github.com/3scale/porta/pull/761/files#diff-b089614cf89e861a94a4d602a70ba0a0R1)